latex2sympy.py

```diff
@@ -301,7 +290,7 @@ def convert_postfix(postfix):
             if ev.eval_at_sub():
                 at_a = do_subs(exp, ev.eval_at_sub())
             if at_b is not None and at_a is not None:
-                exp = sympy.Add(at_b, -1 * at_a, evaluate=False)
+                exp = Sub(at_b, at_a, evaluate=False)
```

Comment:
Without the `in_parsing` argument I think this is behaving the same as `Add`. Not sure if that should need to be passed in though.

Reply:
@tragessere i think you are right, i will try changing it